src/dotnet/project.json

```diff
@@ -8,6 +8,7 @@
         "commands/dotnet-new/CSharp_Lib.zip",
         "commands/dotnet-new/CSharp_xunittest.zip",
         "commands/dotnet-new/CSharp_Web.zip",
+        "commands/dotnet-new/CSharp_nunittest/**"
```

Comment:
You are missing the trailing comma here, thus the build failures, plus this needs to be a Zip like the other items.